### PR TITLE
etcd: 'etcd-member.service' in Container Linux

### PR DESCRIPTION
**Pull request by gyuho**
Fix https://github.com/coreos/etcd/issues/7436.

Also address https://github.com/coreos/docs/issues/989.

/cc @euank @crawford @xiang90

### CODE REVIEW

File: etcd/getting-started-with-etcd.md [hunk 1]
```diff
@@ -44,6 +44,50 @@ etcd v3 startup can be configured at a new node's first boot with [Ignition][ign
...
+    --advertise-client-urls https://10.240.0.1:2379 \
+    --listen-peer-urls https://10.240.0.1:2380 \
+    --initial-advertise-peer-urls https://10.240.0.1:2380 \
+    --initial-cluster my-etcd-1=https://10.240.0.1:2380,my-etcd-2=https://10.240.0.2:2380,my-etcd-3=https://10.240.0.3:2380 \
+    --initial-cluster-token my-etcd-token \
+    --initial-cluster-state new \
+    ...
+EOF
+$ mkdir -p /etc/systemd/system/etcd-member.service.d
+$ mv /tmp/override-my-etcd.conf /etc/systemd/system/etcd-member.service.d/override.conf
```
Comment (lucab) [inline etcd/getting-started-with-etcd.md]: I think we usually name those `nnn-label.conf` to make clear multiple dropins are loaded in lexicographic order. I.e. `10-override.conf` here.

File: etcd/getting-started-with-etcd.md [hunk 2]
```diff
@@ -44,6 +44,50 @@ etcd v3 startup can be configured at a new node's first boot with [Ignition][ign
...
+    --initial-cluster my-etcd-1=https://10.240.0.1:2380,my-etcd-2=https://10.240.0.2:2380,my-etcd-3=https://10.240.0.3:2380 \
+    --initial-cluster-token my-etcd-token \
+    --initial-cluster-state new \
+    ...
+EOF
+$ mkdir -p /etc/systemd/system/etcd-member.service.d
+$ mv /tmp/override-my-etcd.conf /etc/systemd/system/etcd-member.service.d/1-override.conf
+
+# to check service-file-override status
+$ systemd-delta --type=extended
```
Comment (euank) [inline etcd/getting-started-with-etcd.md]: I think `systemctl cat etcd-member` is more readable, but *shrug

File: etcd/getting-started-with-etcd.md [hunk 3]
```diff
@@ -44,6 +44,50 @@ etcd v3 startup can be configured at a new node's first boot with [Ignition][ign
...
+Environment="ETCD_USER=etcd"
+...
+COMMENT
+```
+
+Default configurations can be overwritten with `1-override.conf` file, as below:
+
+```bash
+# to update etcd-member service file
+$ cat > /tmp/override-my-etcd.conf <<EOF
```
Comment (euank) [inline etcd/getting-started-with-etcd.md]: I've always wondered why we do this cat/mkdir/mv song and dance in our docs vs `systemctl edit etcd-member`.

Is there a particular reason here?

Comment (gyuho) [inline etcd/getting-started-with-etcd.md]: No particular reason, I just find it easy to move around those when `sudo` is needed.

**Pull request merged**